extensions.json

```diff
@@ -52,10 +49,11 @@
     "repository": "https://github.com/amir9480/vscode-laravel-extra-intellisense"
   },
   "amodio.tsl-problem-matcher": {
-    "repository": "https://github.com/eamodio/vscode-tsl-problem-matcher"
-  },
-  "andreweinand.mock-debug": {
-    "repository": "https://github.com/microsoft/vscode-mock-debug"
+    "repository": "https://github.com/eamodio/vscode-tsl-problem-matcher",
+    "custom": [
+      "yarn install",
+      "vsce package -o extension.vsix"
+    ]
   },
```

Comment:
I just checked this particular extension, and it seems it's been resulting in an `Internal Server Error` for close to a year now. I will check with the team internally at Eclipse to see what's up. The extension is resolving just fine apparently so I don't think any custom config will be needed.

Reply:
Okay, I'll remove this section then